libs/platform/experience/src/services/layout/plugins/properties/sticky/sticky-layout.plugin.ts

```diff
@@ -13,15 +13,17 @@ export class StickyLayoutPlugin implements LayoutPlugin {
     return ssrAwaiter(import('./sticky.styles').then((m) => m.styles));
   }
 
-  getConfig(): LayoutPluginConfig {
-    return {
+  getConfig(): Observable<LayoutPluginConfig> {
+    return of({
       schema: () => import('./sticky-layout.schema').then((m) => m.schema),
-    };
+    });
   }
 
-  getStyleProperties(data: LayoutStyleProperties): LayoutPluginStyleProperties {
-    return {
+  getStyleProperties(
+    data: LayoutStyleParameters
+  ): Observable<LayoutStyleProperties> {
+    return of({
       'max-height': `calc(${data.height ?? '100vh'} - ${data.top ?? '0px'})`,
-    };
+    });
```

Comment:
I thought we're going to have the ability to resolve the schema properties from this plugin as well as from other plugins.